docker-compose.yml

```
@@ -97,4 +97,4 @@ services:
 networks:
     default:
         name: nigiri
-        external: true
+        external: true
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)